src/pages/Signin/Signin.js

```diff
@@ -1,8 +1,90 @@
 import React from 'react';
 import './Signin.scss';
+import { useNavigate } from 'react-router-dom';
 
 const Signin = () => {
-  return <div>로그인 페이지 입니다.</div>;
+  const navigate = useNavigate();
+
+  function handleButton() {
+    console.log('button 태그 동작');
+    navigate('/');
+  }
+  function handleButton2() {
+    console.log('button 태그 동작');
+    navigate('/signup');
+  }
+
+  return (
+    <body>
+      <div className="header">
+        <div className="headerBox">
+          <div className="inner">
+            <h1 className="pageTitle">로그인</h1>
+            <button
+              className="btnPageClose"
+              type="button"
+              onClick={handleButton}
+            >
+              X
+            </button>
+          </div>
+        </div>
+      </div>
+      <section className="containar">
+        <div className="loginSection">
+          <div className="loginGuide">
+            <span>아이디로 로그인해주세요.</span>
+          </div>
+          <div>
+            <input
+              className="loginInput"
+              type="text"
+              placeholder="아이디 입력"
+            />
+            {/* <button className="delBtn" type="button">
+              x
+            </button> */}
+          </div>
+          <div>
+            <input
+              className="passwordInput"
+              type="password"
+              placeholder="비밀번호 입력(영문,숫자,특수문자 입력)"
+            />
+            {/* <button className="delBtn" type="button">
+              x
+            </button> */}
+          </div>
+          <div>
+            <input className="loginCheck" type="checkbox" />
+            <span>아이디 저장</span>
+          </div>
+          <div>
+            <button className="loginBtn">로그인</button>
+          </div>
+          <ul className="bottomMenu">
+            <li>
+              <a className="idSearch" href="">
```

Comment:
```suggestion
              <Link className="idSearch" to="이동하고싶은 router주소">
```
a 태그는 프로젝트 외부로 보내는 태그 입니다.
프로젝트 내부에서 이동하려면 react-router-dom에서 제공하는 <Link/> 컴포넌트를 써보세요